cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"net/http"
+
+	"k8s.io/klog"
+	controllers "k8s.io/kubernetes/pkg/controller/mizar"
+)
+
+func startMizarStarterController(ctx ControllerContext) (http.Handler, bool, error) {
+	controllerName := "mizar-starter-controller"
+	klog.V(2).Infof("Starting %v", controllerName)
+
+	go controllers.NewMizarStarterController(
+		ctx.InformerFactory.Core().V1().ConfigMaps(),
+		ctx.ClientBuilder.ClientOrDie(controllerName),
+		ctx,
+		startHandler,
+	).Run(2, ctx.Stop)
+	return nil, true, nil
+}
+
+func startHandler(controllerContext interface{}, grpcHost string) {
+	ctx := controllerContext.(ControllerContext)
```

Comment:
It won't. Here in line 39 I have to declare controllerContext as interface{} instead of type ControllerContext. It's because the context will be pass into start controller then pass back when starting other controllers. Inside start controller I cannot import ControllerContext's package to avoid circle import.